cmd/clone.go

```diff
@@ -241,7 +239,7 @@ func getCloneUrls(isOrg bool) ([]scm.Repo, error) {
 
 func createDirIfNotExist() {
 	if _, err := os.Stat(os.Getenv("GHORG_ABSOLUTE_PATH_TO_CLONE_TO") + parentFolder); os.IsNotExist(err) {
-		err = os.MkdirAll(os.Getenv("GHORG_ABSOLUTE_PATH_TO_CLONE_TO"), 0700)
+		err = os.MkdirAll(os.Getenv("GHORG_ABSOLUTE_PATH_TO_CLONE_TO"), 0o700)
```

Comment:
Thanks for the PR! This is a great addition. 

Once question, I'm not sure about this change, is this a valid file permission? I'm curious what this does, I couldn't find anything on it.

Reply:
Oops! So sorry. I was careless here and didn’t realise this was committed. I’ll remove and force push it out.